src/client/components/three/computed.ts

```diff
@@ -45,7 +45,6 @@ const computedInstance = <T, O>(
 const disposedComputed = <T extends { dispose(): void; }>(fn: () => T): IComputedValue<T> => {
   let latest: T | undefined
   const expr = computed(() => {
-    latest && latest.dispose()
```

Comment:
Does this even do anything anymore without dispose?
Now it's basically the same as `computed(fn)` right?

Reply:
The `dispose` on `onBecomeUnobserved` is still used below.

Reply:
improved with c5af825e

Reply:
At this point it's almost worth using atoms for this

Reply:
How would atoms help?

Reply:
Atoms are designed to manage a resource and notify when things change, aren't they? Which is what we are doing here?

Reply:
Fair enough